src/RapidezStatamicServiceProvider.php

```diff
@@ -84,6 +85,7 @@ public function bootRoutes() : self
     public function bootViews() : self
     {
         $this->loadViewsFrom(__DIR__.'/../resources/views', 'rapidez-statamic');
+        $this->loadViewsFrom(__DIR__.'/../resources/views/vendor/responsive-images', 'responsive-images');
```

Comment:
Why do we need this line?

Reply:
Had to specify it coming from `responsive-images` otherwise it wouldn't recognize it.
Had to actually commit the file also.. Did that in the commit below